tests/unit_tests/c_wrappers/res/enkf/test_summary_observation.py

```diff
@@ -16,3 +24,76 @@ def test_std_scaling():
     sum_obs.updateStdScaling(0.50, active_list)
     sum_obs.updateStdScaling(0.125, active_list)
     assert sum_obs.getStdScaling() == 0.125
+
+
+def run_sim(start_date):
+    """
+    Create a summary file, the contents of which are not important
+    """
+    ecl_sum = EclSum.writer("ECLIPSE_CASE", start_date, 3, 3, 3)
+    ecl_sum.addVariable("FOPR", unit="SM3/DAY")
+    t_step = ecl_sum.addTStep(1, sim_days=1)
+    t_step["FOPR"] = 1
+    ecl_sum.fwrite()
+
+
+@pytest.mark.parametrize(
+    "time_delta, expectation",
+    [
+        pytest.param(
+            "1.000347222", does_not_raise(), id="30 seconds offset from 1 day"
+        ),
+        pytest.param(
+            "0.999664355", does_not_raise(), id="~30 seconds offset from 1 day"
```

Comment:
Should the id here be -30 seconds and not ~30 seconds?

Reply:
It was intentionally set to `~`, as it is not exactly -30 seconds

Reply:
Ah I see, thanks.